plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -303,13 +422,17 @@ protected boolean canHandle(Network network, Service service) {
         }
 
         if (!_networkModel.isProviderForNetwork(getProvider(), network.getId())) {
-            s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("NuageElement is not a provider for network " + network.getDisplayText());
```

Comment:
Changed <code>NuageElement</code> to <code>NuageVsp</code> to reflect the name of the provider. No additional context information needed.